faq.rst

```diff
@@ -287,29 +293,49 @@ Then set it as the upstream::
 How do I compare my local repository to a remote repository?
 ''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''
 
-To display the list of changes that are in your local repository, but not
-in the remote, use::
+You have in your clone a copy of the state of the branches in the remote
+repository.  This state is updated any time you do a ``git pull`` or
+``git fetch``.  You can find out if your local copy is up to date with
+its origin by running::
+
+    $ git remote show origin
+
+If there are commits in any of the remote branches that you haven't
+merged into your local copies of the branches, this command will tell
+you so, and also tell you whether or not you can do a fast-forward merge
+(a ``git pull`` without having to make a separate merge commit).
+
+You can do::
+
+    $ git fetch
+
+to make sure your copies of the remote branches are up to date.
```

Comment:
This won't update the current local branch, correct? If so you might want to mention that.